### PR TITLE
Remove obsolete app projects

### PR DESCRIPTION
**Pull request by samsimpson1**
The resources in these projects have either already been deleted or are on a list to be removed in the future.

alphagov/govuk-infrastructure#1127

### CODE REVIEW

File: terraform/projects/app-asset-master/main.tf [hunk 1]
```diff
@@ -1,45 +0,0 @@
...
-  region  = var.aws_region
-  version = "2.46.0"
-}
-
-data "aws_route53_zone" "internal" {
-  name         = var.internal_zone_name
-  private_zone = true
-}
-
-resource "aws_efs_file_system" "assets-efs-fs" {
```
Comment (sengi) [inline terraform/projects/app-asset-master/main.tf:21]: Is this the EFS filer that we're still using for temporarily storing uploaded attachments while they're scanned with Clam? Might need to remember to import it somewhere afterwards :)

File: terraform/projects/app-shared-documentdb/main.tf [hunk 2]
```diff
@@ -1,166 +0,0 @@
...
-    value = var.profiler
-  }
-
-  parameter {
-    name  = "profiler_threshold_ms"
-    value = var.profiler_threshold_ms
-  }
-}
-
-resource "aws_docdb_cluster" "cluster" {
```
Comment (sengi) [inline terraform/projects/app-shared-documentdb/main.tf:130]: I think this one's still in use too so gonna need to import it somewhere at some point.

**Review by sengi — APPROVED** [review]
Awesome! Thanks so much for sorting through all this old cruft!

Spotted a couple of resources that I think we don't wanna delete (but still fine to delete the TF — we can just import those one or two things into govuk-infrastructure somewhere — just pointing 'em out so we don't delete the actual objects)

File: terraform/projects/app-asset-master/main.tf (hunk 1 above)
Comment (samsimpson1) [reply terraform/projects/app-asset-master/main.tf:21]: Ah yes, I thought we were using a new efs volume for this but turns out it's still using the old one. I'll put it on the list of things that need moving across

File: terraform/projects/app-shared-documentdb/main.tf (hunk 2 above)
Comment (samsimpson1) [reply terraform/projects/app-shared-documentdb/main.tf:130]: Oh yeah, this needs adding to the list too. When I looked on the AWS dashboard earlier it wasn't showing any documentdb clusters. Odd

**Pull request merged**